app/scripts/models/account.js

```diff
@@ -605,9 +605,9 @@ define(function (require, exports, module) {
      * Check whether all the passed in permissions have been
      * seen previously.
      *
-     * @param {string} clientId
-     * @param {strings[]} permissions
-     * @returns {boolean} `true` if client has seen all the permissions,
+     * @param {String} clientId
+     * @param {Strings[]} permissions
```

Comment:
I prefer your suggestion, though it goes against the [jsdoc spec](http://usejsdoc.org/tags-type.html):

```

An array of MyClass instances.

{Array.<MyClass>}
// or:
{MyClass[]}
```